larray_editor/editor.py

```diff
@@ -288,6 +284,25 @@ def setup_menu_bar(self):
         file_menu.addSeparator()
         file_menu.addAction(create_action(self, _('&Quit'), shortcut="Ctrl+Q", triggered=self.close))
 
+        #################
+        #   EDIT MENU   #
+        #################
+        edit_menu = menu_bar.addMenu('&Edit')
+
+        #===============#
+        #   UNDO/REDO   #
+        #===============#
+        if qtconsole_available:
+            undo_action = self.edit_undo_stack.createUndoAction(self, "&Undo")
+            undo_action.setShortcuts(QKeySequence.Undo)
+            undo_action.triggered.connect(self.update_title)
```

Comment:
I find it a bit strange that you need to do this explicitly. Shouldn't the update/value change itself trigger the title update?

Reply:
call to `self.edit_undo_stack.canUndo()` in `MappingEditor.unsaved_modifications` will return the expected result once the call to `EditArrayCommand.undo/redo` is finished. 

From line `self.arraywidget.model_data.dataChanged.connect(self.update_title)` in `MappingEditor.init`, `update_title` is automatically called when data changed but this happened inside `undo()` and `redo()`, not after their call.